modules/monitoring-modifying-retention-time-and-size-for-prometheus-metrics-data.adoc

```diff
@@ -41,12 +51,14 @@ metadata:
 data:
   config.yaml: |
     prometheusK8s:
-      retention: <time_specification>
+      retention: <time_specification> <1>
```

Comment:
Line 34
`To modify the retention time for the Prometheus instance that monitors core OpenShift Container Platform projects:`
the example also set retention size, change to
`To modify the retention time and size for the Prometheus instance that monitors core OpenShift Container Platform projects:`

Reply:
fixed